README.md

```diff
@@ -54,9 +264,11 @@ Here some examples for the most useful gRPC APIs that your application can use:
 ### Execute a task
 
 ```javascript
-const MESG = require('mesg-js').application()
+const { application } = require('mesg-js')
+
+const grpc = application().api
```

Comment:
i would recommend to keep `api` instead of `grpc`

Reply:
done